common/reviews/api/heft-webpack5-plugin.api.md

```diff
@@ -0,0 +1,45 @@
+## API Report File for "@rushstack/heft-webpack5-plugin"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { Configuration } from 'webpack-dev-server';
+import type { HeftConfiguration } from '@rushstack/heft';
+import type { HeftSession } from '@rushstack/heft';
+import type { IBuildStageProperties } from '@rushstack/heft';
+import type { IBundleSubstageProperties } from '@rushstack/heft';
+import type { IHeftPlugin } from '@rushstack/heft';
+import * as webpack from 'webpack';
+
+// Warning: (ae-forgotten-export) The symbol "WebpackPlugin" needs to be exported by the entry point index.d.ts
```

Comment:
Seems like this should be exported, or else `_default` should be an `@internal` API

Reply:
Another option would be to change the type from `WebpackPlugin` to be `IHeftPlugin`